packages/rollup-plugin/src/index.js

```diff
@@ -23,6 +23,7 @@ module.exports = function stylexPlugin({
   unstable_moduleResolution = { type: 'commonJS', rootDir: process.cwd() },
   fileName = 'stylex.css',
   babelConfig: { plugins = [], presets = [] } = {},
+  stylexImports = ['stylex', '@stylexjs/stylex'],
```

Comment:
We can probably change this to just `@stylexjs/stylex`.

Reply:
IMHO we could keep `['stylex', '@stylexjs/stylex']` to prevent any potential breaking changes.